operations/setup.go

```diff
@@ -43,7 +44,7 @@ func (c *serviceConf) getSenders(ctx context.Context, conf *model.CedarConfig) (
 
 	if c.interactive {
 		senders = append(senders, send.MakeNative())
-	} else {
+	} else if !c.disableLocalLogging {
```

Comment:
It doesn't look like we're going to hit this at the moment, but if `!c.interactive` and `c.disableLocalLogging` then line 57 would panic.

Reply:
Good catch! Just handled that more explicitly - an error is returned now if no senders were configured and no conf was given.